src/request/change-password/ChangePassword.js

```diff
@@ -0,0 +1,211 @@
+/* global Nimiq */
+/* global Constants */
+/* global PassphraseBox */
+/* global PassphraseSetterBox */
+/* global KeyStore */
+/* global Errors */
+/* global Utf8Tools */
+/* global TopLevelApi */
+/* global DownloadLoginFile */
+/* global LoginFileIcon */
+/* global ProgressIndicator */
+/* global Iqons */
+/* global LoginFile */
+
+/**
+ * @callback ChangePassword.resolve
+ * @param {KeyguardRequest.SimpleResult} result
+ */
+
+class ChangePassword {
+    /**
+     * @param {Parsed<KeyguardRequest.SimpleRequest>} request
+     * @param {ChangePassword.resolve} resolve
+     * @param {reject} reject
+     */
+    constructor(request, resolve, reject) {
+        this._resolve = resolve;
+        this._request = request;
+        this._reject = reject;
+
+        /** @type {Key} */
+        // eslint-disable-next-line no-unused-expressions
+        this._key;
```

Comment:
Could you make this `{Key?}` and assign it `null` instead of the eslint-disable?

Reply:
But then I have to cast it later on, which adds much more noise.

Reply:
But `_key` _is_ `null` until the account is unlocked, so this is even _technically_ wrong.

Reply:
Agreed. Found another solution.